drltrace_src/drltrace_options.cpp

```diff
@@ -77,6 +77,10 @@ droption_t<std::string> op_config_file
  " of library functions for printing library call arguments.  See drltrace documentation"
  " for more details.");
 
+droption_t<std::string> op_filter_file
+(DROPTION_SCOPE_CLIENT, "filter", "filter.config", "The path of the whitelist/blacklist file.",
```

Comment:
It would be great to add this in the README.md and describe this option for anyone who wants to use it.